modules/rhdh-install-web-console.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * layered-products-web-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhdh-install-web-console_{context}"]
+=  Installing the {rh-dev-hub} using the {product-title} web console
+
+The web console provides instructions in a quick start to install the {rh-dev-hub} Operator.
+
+.Prerequisites
+* You must be logged into the {product-title} web console with `admin` privileges.
```

Comment:
The IBM SG says use "logged in to" not "into".

Reply:
It's helpful to provide authors with links @mramendi to guidelines for clarity. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l:~:text=noun%3A%20one%20word.-,log%20in%20to%20verb,-Write%20as%20shown